metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -263,12 +302,16 @@ def build_corp_user_mce(
 
     def build_corp_group_mce(self, attrs: dict) -> Optional[MetadataChangeEvent]:
         """Creates a MetadataChangeEvent for LDAP groups."""
-        cn = attrs.get("cn")
+        cn = attrs.get(self.config.attrs_mapping["cn"])
         if cn:
             full_name = cn[0].decode()
-            owners = parse_from_attrs(attrs, "owner")
-            members = parse_from_attrs(attrs, "uniqueMember")
-            email = attrs["mail"][0].decode() if "mail" in attrs else full_name
+            owners = parse_from_attrs(attrs, self.config.attrs_mapping["owner"])
+            members = parse_from_attrs(attrs, self.config.attrs_mapping["uniqueMember"])
```

Comment:
Remember -> this members field is deprecated. instead, we should be populating the "GroupMembership" aspect of the user object.. Were you intending to do this as a followup?

Reply:
yes, as a part of #3335

Reply:
Wonderful! Thanks for the update